aroquev/src/main/webapp/script.js

```diff
@@ -40,3 +40,36 @@ function calculateArmandoAge() {
     ageContainer.innerText = Math.abs(ageDt.getUTCFullYear() - 1970);
     
 }
+
+// Scripts for the image gallery
+var slideIndex = 1;
+
+function initSlides() {
+  showSlides(slideIndex);
+}
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+// Thumbnail image controls
+function currentSlide(n) {
```

Comment:
Name this function a verb phrase like `updateCurrentSlide(n)` to indicate it performs an action/changes state. The name `currentSlide(n)` makes me think this function will just return what the current slide is.

Reply:
Done.